doc/source/management/audit_log_plugin.rst

```diff
@@ -141,18 +141,229 @@ Ability to stream the audit log to `syslog <http://www.syslog.org/>`_ was implem
+included or excluded from audit logging.
+
+.. warning:: 
+
+  Only one of these variables can contain the list of users to be either
+  included or exclude, while other needs to be ``NULL``. If one of the
+  variables is set to be not ``NULL`` (contains a list of users), the attempt
+  to set another one will fail. Empty string means an empty list.
+
+.. note:: 
+
+  If both :variable:`audit_log_exclude_accounts` and
+  :variable:`audit_log_include_accounts` are ``NULL`` every session will be
+  logged.
+
+Example
+-------
+
+To check the available commands you can run:
+
+.. code-block:: mysql
+
+  mysql> SELECT name FROM performance_schema.setup_instruments WHERE name LIKE "statement/sql/%" ORDER BY name;
+  +------------------------------------------+
+  | name                                     |
+  +------------------------------------------+
+  | statement/sql/alter_db                   |
+  | statement/sql/alter_db_upgrade           |
+  | statement/sql/alter_event                |
+  | statement/sql/alter_function             |
+  | statement/sql/alter_procedure            |
+  | statement/sql/alter_server               |
+  | statement/sql/alter_table                |
+  | statement/sql/alter_tablespace           |
+  | statement/sql/alter_user                 |
+  | statement/sql/analyze                    |
+  | statement/sql/assign_to_keycache         |
+  | statement/sql/begin                      |
+  | statement/sql/binlog                     |
+  | statement/sql/call_procedure             |
+  | statement/sql/change_db                  |
+  | statement/sql/change_master              |
+  ...
+  | statement/sql/xa_rollback                |
+  | statement/sql/xa_start                   |
+  +------------------------------------------+
+  145 rows in set (0.00 sec)
+
+You can add commands to the include filter by running:
+
+.. code-block:: mysql
+
+  mysql> SET GLOBAL audit_log_include_commands= 'set_option,create_db';
+
+If you now create a database:
+
+.. code-block:: mysql
+
+  mysql> CREATE DATABASE world;
+
+You'll see it the audit log:
+
+.. code-block:: none
+
+  <AUDIT_RECORD
+    NAME="Query"
+    RECORD="10724_2016-08-18T12:34:22"
+    TIMESTAMP="2016-08-18T15:10:47 UTC"
+    COMMAND_CLASS="create_db"
+    CONNECTION_ID="61"
+    STATUS="0"
+    SQLTEXT="create database world"
+    USER="root[root] @ localhost []"
+    HOST="localhost"
+    OS_USER=""
+    IP=""
+    DB=""
+  />
+
+You can add commands to a different list by first setting the value of list they are currently in to ``NULL``:
+
+.. code-block:: mysql
+
+  mysql> SET GLOBAL audit_log_include_commands = NULL;
+  Query OK, 0 rows affected (0.00 sec)
+
+  mysql> SET GLOBAL audit_log_exclude_commands= 'set_option,create_db';
+  Query OK, 0 rows affected (0.00 sec)
+
+.. note::
+
+  Invocation of stored procedure has ``sql_command`` field ``call_procedure``. All statements within the procedure have the ``sql_command`` ``call_procedure`` as well.
```

Comment:
Invocation of stored procedures have command type call_procedure, and all the statements executed within the procedure have the same type call_procedure as well.